### PR TITLE
DomConverter.domToView() will return null for comments.

### LINKED ISSUE #647: DomConverter throws an error when found <!-- comment -->
[This line](https://github.com/ckeditor/ckeditor5-engine/blob/master/src/view/domconverter.js#L309) throws an error: `Uncaught TypeError: Cannot read property 'toLowerCase' of undefined` when `domNode` is a [`Comment`](https://developer.mozilla.org/en-US/docs/Web/API/Comment).

I figured out it during our new testing environment (Karma & Webpack). Karma inserts a comment to `body` tag. `DomConverter` assumes that found `domNode` is an instance of `HTMLElement`.

For the first iteration, `domNode` is a `document.body`. For the second iteration, `domNode` is an instance of `Comment`.

image

I also reproduced this error in Bender. 

Steps to reproduce:
1. Add to this file - [`tests/view/observer/__template__.html`](https://github.com/ckeditor/ckeditor5-engine/blob/317e1a1adf2674277e70413746f081b8700de4ae/tests/view/observer/__template__.html) - some HTML comment: `<!-- some comment -->`
2. Run the test: [`tests/engine/view/observer/domeventdata`](http://tests.ckeditor.dev:1030/build/modules/amd/tests/engine/view/observer/domeventdata)

I know this issue is like "edge case" but we should prevent it.

### PR DESCRIPTION
**Pull request by Reinmar**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: `DomConverter#domToView()` will not throw when converting a comment. Closes #647.

### CODE REVIEW

**Comment by Reinmar** [comment]
image

This is enough for now.

File: tests/view/domconverter/domconverter.js [hunk 1]
```diff
@@ -69,48 +69,65 @@ describe( 'DomConverter', () => {
...
-		describe( 'isElement', () => {
+		describe( 'isElement()', () => {
 			it( 'should return true for HTMLElement nodes', () => {
 				expect( converter.isElement( element ) ).to.be.true;
 			} );
 
 			it( 'should return false for other arguments', () => {
 				expect( converter.isElement( text ) ).to.be.false;
 				expect( converter.isElement( documentFragment ) ).to.be.false;
+				expect( converter.isText( comment ) ).to.be.false;
```
Comment (scofalik) [inline tests/view/domconverter/domconverter.js]: `isElement`

File: tests/view/domconverter/domconverter.js [hunk 2]
```diff
@@ -69,48 +69,65 @@ describe( 'DomConverter', () => {
...
-		describe( 'isDocumentFragment', () => {
+		describe( 'isDocumentFragment()', () => {
 			it( 'should return true for HTMLElement nodes', () => {
 				expect( converter.isDocumentFragment( documentFragment ) ).to.be.true;
 			} );
 
 			it( 'should return false for other arguments', () => {
 				expect( converter.isDocumentFragment( text ) ).to.be.false;
 				expect( converter.isDocumentFragment( element ) ).to.be.false;
+				expect( converter.isText( comment ) ).to.be.false;
```
Comment (scofalik) [inline tests/view/domconverter/domconverter.js]: You sure tried hard to have `isText` throughly tested ;)
